proposals/wgs/training/wg-charter-training.md

```diff
@@ -0,0 +1,37 @@
+# WG Training Charter
+
+This charter adheres to the conventions, roles and organization management outlined in [wg-governance].
+
+## Scope
+
+WG Training covers developing, deploying, and operating training jobs on Kubeflow.
+
+### In scope
+
+#### Code, Binaries and Services
+
+- APIs used for running distributed training jobs (e.g., TFJob API)
+- Tools and documentation to aid in ecosystem tool interoperability around distributed training jobs (e.g., TFJob CRD/Controller)
+
+#### Cross-cutting and Externally Facing Processes
+
+- Coordinating with WG Pipeline to make sure distributed training jobs can interact well with pipelines
+- Coordinating with release teams to ensure that the distributed training features can be released properly
+- Coordinating with WG AutoML to integrate with AutoML systems (e.g. Katib)
+
+### Out of scope
+
+- APIs used for running inference/serving tasks (this falls under the purview of WG Serving).
```

Comment:
What about hyper parameter tuning?

Reply:
/cc @andreyvelich @jlewi 

We are still considering if we should have WG AutoML. Thus did not add it here.